catkit2/services/hamamatsu_camera/hamamatsu_camera.py

```diff
@@ -270,8 +261,90 @@ def monitor_temperature(self):
             temperature = self.get_temperature()
             self.temperature.submit_data(np.array([temperature]))
 
+            if temperature > 30:

```

Comment:
The limiting temperature should be something that is read from the config.

Reply:
currently need to be tested if this is the right number